.circleci/docker/requirements-ci.txt

```diff
@@ -0,0 +1,214 @@
+# Python dependencies required for unit tests
+
+#awscli
+#Description: AWS command line interface
+#Pinned versions: 1.6
```

Comment:
Does this mean that it's pinned elsewhere? 
Why do we not already write 
awscli==1.6
if there's only one pin?

Reply:
There is only one pin, 1.6.  The awscli is currently commented out.  I am still trying to get the install to work without errors on some platforms